README.md

```diff
@@ -1,2 +1,38 @@
+---
+title: "CFSS hw01"
+author: "Yanning Cui"
+date: "January 9, 2017"
+output: pdf_document
+---
```

Comment:
PDF output is one option for R Markdown documents. Though in this context it actually is unnecessary. Markdown documents are automatically rendered on GitHub.com, so you only need to output it to PDF or HTML if it is to be viewed elsewhere. We'll see this soon enough on hw02